src/source-control/GitSourceControl.ts

```diff
@@ -310,7 +320,6 @@ export class GitSourceControl implements SourceControl {
         tree,
         [targetNodegitCommit],
       );
-
```

Comment:
Hmm, should we add these empty lines back? I think they help to paragraph the code better so that it doesn't look like one giant chunk

Reply:
Readded, as discussed on Discord 🎉